tsl/src/nodes/decompress_chunk/decompress_chunk.h

```diff
@@ -92,6 +92,12 @@ typedef struct DecompressChunkPath
 	 */
 	bool perform_vectorized_aggregation;
 
+	/*
+	 * Columns that are used for vectorized aggregates. The list contains for each attribute -1 if
+	 * this is not an vectorized aggregate column or the Oid of the data type of the attribute.
+	 */
+	List *vectorized_aggregation_column;
+
```

Comment:
Might be good to name it something like "aggregated_column_type" to make more clear what it contains, and move here the comment about why we can't determine the column type form the resulting tupdesc.

Probably we have to switch to proper separation of aggregated tupdesc and decompressed tupdesc at some point. Might be easier to do it with a separate aggregation node, which might also simplify our planning in other ways -- not sure.

Reply:
Renamed and moved the comment as suggested.